contributors.txt

```diff
@@ -1 +1,2 @@
 MisaKondo
+cocodahlia
```

Comment:
Looks Good To Me.

:beers: :beers: :beers: :beers: :beers: :beers: :beers: :beers: